osf/migrations/0212_add_default_registration_provider.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-07-10 17:34
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from osf.models import RegistrationProvider
+
+
+def add_provider(state, *args, **kwargs):
+    provider, created = RegistrationProvider.objects.get_or_create(_id=RegistrationProvider.default__id)
```

Comment:
I don't believe this migration is necessary. 0138_ensure_subjects_and_providers.py already creates an OSF Registry Provider, so it should already be available. Is there anything else happening here, or is this just to ensure that there's a registration provider with the `_id` of 'osf'?

Reply:
`0138_ensure_subjects_and_providers` didn't seem to work reliably in Travis, I remember finding this migration necessary, but it's possible my local db behind or something.